wx/lib/agw/aui/framemanager.py

```diff
@@ -98,10 +98,10 @@
 
 import wx
 # just for isinstance
-try:
-    from time import time, process_time as clock
-except ImportError:
-    from time import time, clock
+import time
+import sys
+if sys.version_info[0] == 2:  # hotpatch clock which is removed in py3.8
+    time.perf_counter = time.clock
```

Comment:
Let's factor out the common bits and change the critical part of this to something like:

```
try:
    from time import perf_counter
except ImportError:
   from time import clock as perf_counter
```

I'd rather not modify a standard lib module if we don't need to.